### DISCUSSION TITLE
making repositioning of context tag and project tag optional

### DISCUSSION THREAD

**Discussion by Tobida** [todotxt/todo.txt#61] [Q&A]
I love to use todo.txt in a natural way of language:

> discuss with @George about +superproject concerning the aspect of unicorns

This is in consensus with the todo.txt rules i read: "Contexts and Projects may appear anywhere in the line after priority/prepended date"

In my sleek this will show up like this, the sentence is destroyed 

> discuss with about concerning the aspect of unicorns @George +superproject

I fully respect that in your perspective this is not a bug but a feature. 

I would like to use it without the feature of putting tags at the and of the line - perhaps there could be an option  to switch this behaviour off?

Thanks!

**Comment by bobwhitelock** [comment]
I also sometimes write tasks like this, and agree this seems compliant with the spec. I've also encountered programs working with todotxt format rewriting tasks this way - I think this is done as it's easier when parsing things to blow away the user's formatting and then serialise tasks in a consistent way, rather than more intelligently trying to preserve the formatting.

Sounds like this is an implementation issue with the program you're using to work with todotxt tasks though, so you probably want to log an issue there - possibly here: https://github.com/ransome1/sleek?

**Comment by Tobida** [comment]
Argh. Sorry. Wrong project - i was switching to your project to check the todo.txt format rules, but wanted to drop the issue in the sleek-Project ;)
